packages/graphql/src/api-v6/schema-generation/schema-types/filter-schema-types/TopLevelFilterSchemaTypes.ts

```diff
@@ -49,16 +56,32 @@ export class TopLevelFilterSchemaTypes extends FilterSchemaTypes<ConcreteEntity>
         });
     }
 
-    protected get nodeWhere(): InputTypeComposer {
-        return this.schemaBuilder.getOrCreateInputType(this.entity.typeNames.nodeWhere, (itc: InputTypeComposer) => {
+    public get nodeWhere(): InputTypeComposer {
+        return this.schemaBuilder.getOrCreateInputType(this.entityTypeNames.nodeWhere, (itc: InputTypeComposer) => {
             return {
                 fields: {
                     AND: itc.NonNull.List,
                     OR: itc.NonNull.List,
                     NOT: itc,
                     ...this.convertAttributesToFilters([...this.entity.attributes.values()]),
+                    ...this.convertRelationshipsToFilters([...this.entity.relationships.values()]),
                 },
             };
         });
     }
+
+    private convertRelationshipsToFilters(relationships: Relationship[]): Record<string, InputTypeComposer> {
+        const fields: ([string, InputTypeComposer | GraphQLScalarType] | [])[] = filterTruthy(
```

Comment:
Maybe it is more readable with Array?
```suggestion
        const fields: Array<[string, InputTypeComposer | GraphQLScalarType] | []> = filterTruthy(
```

Reply:
No problem!